frontend/src/pages/Analytics/ProjectAnalytics.js

```diff
@@ -213,18 +213,21 @@ const NumberChart = ({ title, budget, currency, dataTest }) => (
     </CardContent>
   </Card>
 );
-const RatioChart = ({ title, budget, dataTest }) => (
-  <Card style={dashboardStyles.card}>
-    <CardContent style={dashboardStyles.ratioContent}>
-      <Typography style={{ flex: 1 }} variant="overline">
-        {title}
-      </Typography>
-      <Typography style={{ flex: 1 }} data-test={dataTest} variant="h6">
-        {budget ? `${(budget * 100).toFixed(2)}%` : "-"}
-      </Typography>
-    </CardContent>
-  </Card>
-);
+const RatioChart = ({ title, budget, dataTest }) => {
```

Comment:
is this frontend change necessary for the api npm package updates?

Reply:
No but since it is a small fix in an extra commit, I see no problem